TravelFunds/2018.md

```diff
@@ -24,21 +24,21 @@ Hassan	|	Sani	|	Collab Summit Spring 2018	|	Attendance & Code & Learn	|	Berlin,
 Richard	|	Littauer	|	Collab Summit Spring 2018	|	Attendance & Collaborate	|	Berlin, Germany	|	31May - 3Jun 2018	|	US$800	|		|		|		|		|		|		|	
 Dan	|	Shaw	|	Spring 2018 Collaborator's Summit and Code & Learn	|	Attendance & WG facilitator	|	Berlin, Germany	|	30May - 3Jun 2018	|	US$1500	|		|		|		|		|		|		|	
 Ben	|	Michel	|	Node.js at MSBuild 2018	|	Node.js Collaboration & CommComm Representation	|	Seattle Washington, USA	|	6May - 9May 2018	|	US$402	|		|		|		|		|		|		|	
-Nicola	|	Del Gobbo	|	Collab Summit Spring 2018	|	Attendance	|	Berlin, Germany	|	30May - 2Jun 2018	|	US$534.47	|		|		|		|		|		|		|	
+Nicola	|	Del Gobbo	|	Collab Summit Spring 2018	|	Attendance	|	Berlin, Germany	|	30May - 2Jun 2018	|	€391.01	|		|		|	26Jun 2018	|	€391.01	|		|		|		|
 Gus	|	Caplan	|	TC39 Meeting	|	Attendance	|	Redmond, WA, US	|	 23Jul –  Jul27 2018	|	US$1112.28	|		|		|		|		|		|		|	
 Dhruv	|	Jain	|	Node Collab Summit & JS Interactive 2018	|	Attendance, Collaborate and Code & Learn	|	Vancouver BC, CAN	|	10Oct - 13Oct 2018	|	US$2100	|		|		|		|		|		|		|	
 Никита	|	Сковорода	|	Collab Summit 2018	|	Attendance & Collaboration	|	Vancouver BC, CAN	|	12Oct - 13Oct 2018	|	US$1500	|		|		|		|		|		|		|	
 Hassan Yabagi Sani	|		|	Collab Summit & JS Interactive 2018	|	Code and Learn	|	Vancouver BC, CAN	|	10Oct - 13Oct 2018	|	US$2376	|		|		|		|	US$2741	|		|		|	US$2741
-Ahmad	|	Bamieh	|	Collab Summit & JS Interactive 2018	|	Attendance, Collaborate and Code & Learn	|	Vancouver BC, CAN	|	10Oct - 13Oct 2018	|	US$2250	|		|		|		|		|		|		|	
-Patricia	|	Realini	|	Node Summit	|	Panel Speaker	|	San Francisco, CA, USA	|	23Jul - 25Jul 2018	|	US$1150	|		|		|		|		|		|		|	
-Alejandro	|	Oviedo	|	Collab Summit & JS Interactive 2018	|	Attendance & Collaborate	|	Vancouver, BC, CAN	|	10Oct - 14Oct 2018	|	US$2200	|	1Jul 2018	|	https://github.com/nodejs/admin/pull/174	|		|		|		|		|	
+Ahmad	|	Bamieh	|	Collab Summit & JS Interactive 2018	|	Attendance, Collaborate and Code & Learn	|	Vancouver BC, CAN	|	10Oct - 13Oct 2018	|	US$2250	|		|		|	20Sep 2018	|	US$2250	|		|		|		|
+Patricia	|	Realini	|	Node Summit	|	Panel Speaker	|	San Francisco, CA, USA	|	23Jul - 25Jul 2018	|	US$1150	|		|		|	17Sep 2018	|	US$ 1083	|		|		|		|
+Alejandro	|	Oviedo	|	Collab Summit & JS Interactive 2018	|	Attendance & Collaborate	|	Vancouver, BC, CAN	|	10Oct - 14Oct 2018	|	US$2200	|	43282	|	https://github.com/nodejs/admin/pull/174	|		|		|		|		|		|
```

Comment:
`43282`typo in date column? Only really noticeable in rich diff mode.

![image](https://user-images.githubusercontent.com/16782063/45907856-4c38e380-bdbf-11e8-8049-fdc3eea3bc8c.png)